### ISSUE TITLE
Feature Request: ethstats support

### ISSUE THREAD

**Issue by MysticRyuujin** [paradigmxyz/reth#4869]
### Describe the feature

It would be cool to get reth to support ethstats, example: [ethstats.dev](https://ethstats.dev/)

Every other client supports the ethstats flags, but it doesn't look like reth does yet.

**Comment by supernovahs** [comment]
Can u give me a little more context ? Maybe with geth.
Thx

**Comment by quickchase** [comment]
https://geth.ethereum.org/docs/monitoring/ethstats#ethstats

**Comment by quickchase** [comment]
Bad bot

**Comment by mattsse** [comment]
wdyt @shekhirin?
I think we want this, can we start with a tracking issue for todos?

**Comment by shekhirin** [comment]
yeah, I see that most of the clients support that, but don't understand if https://ethstats.dev is maintained and used by anybody. IMO it's a nice-to-have, but not high priority, so we can indeed create a tracking issue with TODOs.

**Comment by quickchase** [comment]
ethstats.dev is just a semi public one ran by the ef devops team, the ethstats server is open source and sometimes ran privately as well.

It's just a nice to have

**Comment by xrchz** [comment]
I would like this

**Comment by ohko4711** [comment]
> ethstats.dev is just a semi public one ran by the ef devops team, the ethstats server is open source and sometimes ran privately as well.
> 
> It's just a nice to have

reth is ready for production, the current phase is nice to have ethstats now?

**Comment by onbjerg** [comment]
It's still not a priority for us, but if someone wants to contribute an ethstats reporting task they should feel free. As far as I can tell the API for the ethstats server is undocumented, and it's still somewhat unclear if this is maintained or not:)

**Comment by jenpaff** [comment]
seems like a low-hanging fruit , will look into scoping this 

```
geth <other commands> --ethstats node1:secret:127.0.0.1:9000
```
